sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/ScalaUDF.scala

```diff
@@ -1188,6 +1188,8 @@ case class ScalaUDF(
 
   private[this] val resultConverter = catalystConverter
 
+  private def functionName = udfName.map { uName => s"$uName in $funcCls" }.getOrElse(funcCls)
```

Comment:
Sounds strange that the function is in class name? Maybe 
```suggestion
  private def functionName = udfName.map { uName => s"$uName ($funcCls)" }.getOrElse(funcCls)
```